src/parser/set/set.spec.js

```diff
@@ -0,0 +1,16 @@
+const assert = require('assert');
+const { SET } = require('../../commands');
+const parseSetCommand = require('./set');
+
+describe('parseSetCommand', function() {
+    it('should correctly split the set command into standup name and url', function() {
+        const command = 'set url for artris to artris.com';
+        const expected = {
+            type: SET,
+            who: 'artris',
```

Comment:
who here should be which event. we can find out who from the slash command metadata sent by slack